src/main/java/org/folio/linked/data/controller/advice/ApiExceptionHandler.java

```diff
@@ -88,6 +89,18 @@ public ResponseEntity<ErrorResponse> handleMissingServletRequestParameterExcepti
     return genericBadRequestMapper.errorResponseEntity(e);
   }
 
+  @ExceptionHandler(ValidationException.class)
+  public ResponseEntity<ErrorResponse> handleValidationException(ValidationException exception) {
+    return handleValidationExceptionCause(exception);
+  }
+
+  private ResponseEntity<ErrorResponse> handleValidationExceptionCause(ValidationException exception) {
```

Comment:
We do not need this private method right?Better to include this code in the public handleValidationException method itself?